src/components/chat-view-container/chat-view.test.tsx

```diff
@@ -220,15 +223,31 @@ describe('ChatView', () => {
     expect(wrapper.find(`.${className}`).exists()).toBe(true);
   });
 
-  it('renders skeleton if messages have not been loaded yet', () => {
-    const wrapper = subject({ messages: MESSAGES_TEST, hasLoadedMessages: false });
+  it('renders skeleton if messages are being loaded', () => {
```

Comment:
This doesn't sound right. This means that every time we switch conversations we'll re-show the skeleton again, I think. We definitely don't want that. If the messages have been loaded previously we just want to render them immediately. That's one thing that gives the illusion speed.

Reply:
I had missed that we're doing that. Updated the code to only show the failure message ONLY If the message fetch has failed.